File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-20/source/2026-07-20-reliability-runbook-alert-triage-automation-2.ts
// Northstar Transit Cloud | Checkout SVC-17 | 2026-07-13 | error budget 2.4% | INC-260713
export const topic = "alert_triage";
export const fingerprint = "6484d89d504aa1d7";
export const alertLevel = (depth) => depth >= 24000 ? 'critical' : depth >= 12000 ? 'warning' : 'normal';

File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-20/source/2026-07-20-reliability-runbook-alert-triage-automation-2.rs
// Northstar Transit Cloud | Checkout SVC-17 | 2026-07-13 | error budget 2.4% | INC-260713
pub const TOPIC: &str = "alert_triage";
pub const FINGERPRINT: &str = "38c0c83315c3170b";
pub fn q07_is_critical(depth: u32) -> bool { depth >= 24_000 }
